src/test/java/com/wikia/webdriver/elements/oasis/components/templatecontent/TemplateContent.java

```diff
@@ -0,0 +1,14 @@
+package com.wikia.webdriver.elements.oasis.components.templatecontent;
+
+import com.wikia.webdriver.common.core.api.ArticleContent;
+
+public class TemplateContent {
+
+  private static final String namespace = "Template:";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00115)